juju-tutorials/juju-tricks.md

```diff
@@ -143,9 +226,9 @@ mongo --sslAllowInvalidCertificates --authenticationDatabase admin --ssl -u $(su
 
 ### Manual change of Openstack endpoint certificates
 
-In case you are running juju controller on top of Openstack and for some reason the CA certificates on the endpoints change, you can't access the endpoint anymore with "certificate signed by unknown authority", the only way to fix it is to update the juju mongodb directly.
+In case you are running a juju controller on top of the Openstack and for some reason the CA certificates on the endpoints change, you can't access the endpoint anymore with "certificate signed by unknown authority", the only way to fix it is to update Juju's mongodb directly.
```

Comment:
"on top of OpenStack" (sorry if I gave a wrong correction last time)

Reply:
no, this was my change, I thought the article belongs there.